### PR TITLE
Update to androidx support libraries

### PR DESCRIPTION
**Pull request by Epreuve**
Depends on https://github.com/lkorth/device-automator/pull/3

Update to androidx support libraries.

### CODE REVIEW

File: DeviceAutomator/build.gradle [hunk 1]
```diff
@@ -5,16 +5,16 @@ apply plugin: 'signing'
 version = '0.3.1-SNAPSHOT'
 
 android {
-    compileSdkVersion 26
-    buildToolsVersion '26.0.0'
+    compileSdkVersion 28
```
Comment (quinnjn) [inline DeviceAutomator/build.gradle]: `.travis.yml` will need an update to SDK 28 as well

**Comment by lkorth** [comment]
This can be rebased against master now to drop the SDK and gradle upgrades from this branch since they were merged into master.

Breaking changes are fine on 0.x versions, but I'm fine creating a branch and final release of 0.x and then merging this and creating a 1.0.0.

**Comment by Epreuve** [comment]
Rebase is done 👍  I'm not partial to either versioning approach, so either is fine with me.

**Comment by lkorth** [comment]
[1.0.0](https://github.com/lkorth/device-automator/releases/tag/1.0.0) was just released.

**Pull request merged**